README.md

```diff
@@ -16,4 +16,50 @@ You'll be able to create, edit, and delete links and create a landing page that
 ## Tech stack
 Django
 
-Contribute to the page if you need to by cloning this project, raise an issue, or submit a PR.
\ No newline at end of file
+## How to upload Django Project to Render.com
+
+#### 1. Visit the render.com website and make an account.
+#### 2. Select PostgreSQL from the New+ dropdown.
+![image](https://github.com/Terieyenike/linktree/assets/57314456/a4519ecf-14b2-4799-a5fc-a7c5ef5e4582)
+
+#### 3. Give a name to that database and then create the database
+#### 4. Click on Connect and copy the External Database URL
+#### 5. Now go into your project and in settings.py file, download and import dj_database_url
```

Comment:
I think this is supposed to be `dj-database-url`

https://pypi.org/project/dj-database-url/

Reply:
actually while downloading it it should be pip install dj-database-url and while importing in the code It should be dj_database_url. it is written in the documentation also
https://pypi.org/project/dj-database-url/

Reply:
Please review my PR once again #4